crates/matrix-sdk/src/widget/permissions.rs

```diff
@@ -37,10 +40,154 @@ pub struct Permissions {
     pub read: Vec<EventFilter>,
     /// Types of the messages that a widget wants to be able to send.
     pub send: Vec<EventFilter>,
-    /// If this permission is requested by the widget, it can not operate
-    /// separately from the matrix client.
-    ///
-    /// This means clients should not offer to open the widget in a separate
-    /// browser/tab/webview that is not connected to the postmessage widget-api.
-    pub requires_client: bool,
+    /// Whether the client can open the widget in a separate window.
+    pub can_open_in_separate_window: bool,
```

Comment:
Since the upstream capability is called `io.element.requires_client`, I'd rather keep the previous field name / meaning. Also, the new comment discards much of the clarity we had before.

Reply:
Alright, makes sense. Updated.